sys/mips/conf/std.CHERI_PURECAP

```diff
@@ -0,0 +1,50 @@
+#
+# CHERI -- Kernel for the SRI/Cambridge "CHERI" (Capability Hardware Enhanced
+# RISC Instructions) FPGA soft core, as configured in its Terasic DE-4
+# reference configuration - 128bit version.
+#
+# $FreeBSD$
+#
+
+include		"std.CHERI"
+
+machine		mips mips64c128
+
+nomakeoptions 	ARCH_FLAGS
+makeoptions 	ARCH_PIC_FLAGS="-fpic"
+makeoptions 	ARCH_FLAGS="-mcpu=cheri128 -mxcaptable"
+makeoptions 	CFLAGS+="-gdwarf-5"
+makeoptions 	WERROR="-Wno-error"
```

Comment:
Should this not be in the .mk files?

Reply:
I moved most of those. I left WERROR as we would ideally remove it